docs/blog/2018-10-09-hacktoberfest-kickoff/index.md

```diff
@@ -2,7 +2,7 @@
 title: "Celebrate Hacktoberfest with Gatsby!"
 date: 2018-10-09
 author: "Amberley Romo"
-tags: ["hacktoberfest", "open source"]
```

Comment:
Open source makes sense to keep as a tag, in my opinion, to help us explore the line between OSS contributions and support of funded products. People obviously care a lot about that distinction and we need to do a better job as a company of explaining our philosophy on it–so tagging content as "open source" provides a low-effort way for us to do that. I also think it's confusing to tag this as "gatsby-inc" as it blurs the line too much.